pheweb/serve/data_access/db.py

```diff
@@ -633,6 +640,23 @@ def get_variant_annotations(self, id_list):
             for anno in annotation["hits"]["hits"]
         ]
 
+class MissingVariantDao(MissingVariantDB):
+    def __init__(self, missing_variant_path):
+        self.missing_variant_path = missing_variant_path
+        self.tabix_file = pysam.TabixFile(self.missing_variant_path, parser=None)
+        self.headers = self.tabix_file.header[0].split("\t")
+
+    def get_missing_variant(self, variant: Variant):
+        header = [h.lower() for h in self.headers]
+        for row in self.tabix_file.fetch(f"chr{variant.chr}", start=variant.pos - 1, end=variant.pos + 1):
+            splited_row = row.split("\t")
+            if len(header) == 26 & len(splited_row) == 26:
+                json_obj = dict(zip(header, splited_row))
+                return json_obj
+            else:
+                return None
+        return None,
```

Comment:
This now returns a tuple containing a single None, i.e. `(None,)`, which probably is not what is intended here. remove the comma for it to return just None instead.